src/components/designSystem/Filters/utils.ts

```diff
@@ -7,7 +7,9 @@ export const formatFiltersForInvoiceQuery = (searchParams: URLSearchParams) => {
 
   return Object.entries(filtersSetInUrl).reduce(
     (acc, cur) => {
-      let [key, value] = cur as [AvailableFiltersEnum, string | string[] | boolean]
+      const current = cur as [AvailableFiltersEnum, string | string[] | boolean]
```

Comment:
Oh we cannot destructure like that anymore ? 🤔

Reply:
We can, but we can't use `let` for key as it's a constant. Do you know an other way of writing it?